### PR TITLE
final project

### PR DESCRIPTION
**Pull request by w4xxwrld**
what was done:
1. created postman collection
2. code tested on digital ocean server on api.shakhnazar.online
3. endpoints tested on postman(see loom video in skool)

### CODE REVIEW

File: back-end-102/final-project/src/main.py [hunk 1]
```diff
@@ -1,18 +1,290 @@
...
+
+projects = []
+
+tasks = []
+
+active_refresh_tokens = {}
+
+oauth2_scheme = OAuth2PasswordBearer(tokenUrl="/auth/login")
+
+SECRET_KEY = "BaVArIanMotORWorKs3PoInT5LIterTwINTUrBo"
```
Comment (aissayev) [inline back-end-102/final-project/src/main.py]: ideally should get the secret key from `.env` file, so it is not exposed in the repo

Comment (aissayev) [reply back-end-102/final-project/src/main.py]: .env should be gitignored

File: back-end-102/final-project/src/main.py [hunk 2]
```diff
@@ -1,18 +1,290 @@
...
+    description: Optional[EmailStr] = None
+    status: Optional[str] = None
+    project_id: Optional[int] = None
+
+class Task(BaseModel):
+    id: int
+    title: str
+    description: str
+    status: str  # "Pending", "In Progress", "Completed"
+    project_id: int
```
Comment (aissayev) [inline back-end-102/final-project/src/main.py]: Put classes into a separate file and import it

**Review by aissayev — CHANGES_REQUESTED** [review]

**Comment by w4xxwrld** [comment]
https://www.loom.com/share/d0c5d664b5e54ef9bfb0a67421b43068
here's the video with demonstration of fixes and file moves.
commit with all the changes pushed

**Review by aissayev — COMMENTED** [review]
lgtm

**Pull request merged**